api/clients/eigenda_client.go

```diff
@@ -66,8 +67,8 @@ func (m EigenDAClient) GetBlob(ctx context.Context, BatchHeaderHash []byte, Blob
 		return nil, fmt.Errorf("blob has length zero")
 	}
 
-	version := BlobEncodingVersion(data[0])
-	codec, err := BlobEncodingVersionToCodec(version)
+	version := codecs.BlobEncodingVersion(data[0])
```

Comment:
the codecBlobHeader has now changed, it's a 32 byte value that isn't transformed, the first byte is a 0, second byte is a version byte, and 3rd to 10th bytes are the length encoded as a uint64 to avoid this.

Reply:
26a95912051997b8972d18664c7b76777970fff0

Reply:
following my question, even though the first several bytes are 0, but after taking IFFT, those are very likely non-zero. Mostly likely it will produce an error, because EncodingVersion is wrong. So my second response just says they probably have to know the encodingVersion at first, or you try both encodingVersion

Reply:
so now we don't do an IFFT on the first 32 bytes, only the rawData itself

https://github.com/Layr-Labs/eigenda/blob/9c004ddbb0129f766ea6809915327cbf044ad6eb/api/clients/codecs/ifft_blob_encoding_codec.go#L26